### ISSUE TITLE
[Doc] Sampler: "store" vs "storechain"

### ISSUE THREAD

**Issue by laszukdawid** [dfm/emcee#249]
**General information:**

- emcee version: 2.2.1
- platform: 
- installation method (pip/conda/source/other?):

**Problem description:**

### Expected behavior:
Some of the API documentation is out of date. For example (below), sampler EnsembleSampler used to have attribute "storechain" and now it's "store".

### Actual behavior:
Up to date documentation.

### What have you tried so far?:
Figuring out where the build is made for the documentation, as files in this repo seems to be updated.

### Minimal example:
http://emcee.readthedocs.io/en/stable/user/advanced.html#incrementally-saving-progress
http://dfm.io/emcee/current/api/#emcee.EnsembleSampler.chain
http://emcee.readthedocs.io/en/stable/_modules/emcee/ensemble.html?highlight=storechain

**Comment by andyfaff** [comment]
Dawid, the codebase is going through a bit of churn at the moment in preparation for the next release. There are some major changes in the works, and the documentation may not have kept up. If the docstrings aren't matching the behaviour thats worth reporting.

**Comment by dfm** [comment]
Thanks! The emcee 2.2.1 docs are here http://emcee.readthedocs.io/en/stable/ and the docs for the upcoming release (currently on the master branch) are here: http://emcee.readthedocs.io/en/latest/ It looks to me like the docs are consistent - do you see another problem?

**Comment by laszukdawid** [comment]
Thanks both, @andyfaff and @dfm for explanation. I just noticed that v3 is coming up.

Nevertheless, if /stable/ is meant to reflect v2.2.1 then links I provided, or those below, refer to the previous API and not for v2.2.1. Isn't that just a matter of building docs and replacing them in ReadTheDocs?

Both links below refer to `storechain` where actual api uses `store`. I'm not strongly advocating for changes, but until the update of docs, this might be a source of confusion for new users.
http://emcee.readthedocs.io/en/v2.2.1/user/advanced/
http://emcee.readthedocs.io/en/v2.2.1/api/#emcee.EnsembleSampler
